x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -46,20 +52,21 @@ export interface CustomRule {
   description: string;
   dataSource: RuleDataSource;
   interval?: string;
-  severity: string;
-  riskScore: string;
-  tags: string[];
+  severity?: string;
+  riskScore?: string;
+  tags?: string[];
   timelineTemplate?: string;
-  referenceUrls: string[];
-  falsePositivesExamples: string[];
-  mitre: Mitre[];
-  note: string;
-  runsEvery: Interval;
-  lookBack: Interval;
-  timeline: CompleteTimeline;
-  maxSignals: number;
+  referenceUrls?: string[];
+  falsePositivesExamples?: string[];
+  mitre?: Mitre[];
+  note?: string;
+  runsEvery?: Interval;
+  lookBack?: Interval;
```

Comment:
What's the difference between `interval?: string;` and `runsEvery?: Interval;`?

Reply:
Honestly, I don't know, as everybody is collaborating on adding more Cypress tests, these interfaces have been extended and modified to fulfill other test requirements.

Reply:
Gotcha, let's revisit this later and create a ticket for it.